requirements_dev.txt

```diff
@@ -1,6 +1,6 @@
 attrs>=21.4.0
 black>=21.12b0
-ccbuilder==0.0.5
+ccbuilder==0.0.8
```

Comment:
`0.0.9`?

Reply:
I'll update all versions (including diopter) once this PR and the other repos have stabilized.